### PR TITLE
QA Observation bugfix/FOUR-13084: Requests are not refreshed when switching between My Requests/My Task tabs in Launch Pad

### PR DESCRIPTION
**Pull request by CarliPinell**
## How to Test
-  Open Any Process with "Open in Launchpad" 
-  Create a new request in other window
- Change from tab Request to Task and vice versa

## Related Tickets & Packages
https://processmaker.atlassian.net/browse/FOUR-13084

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

ci:deploy
ci:next

### CODE REVIEW

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-c95a3be734.eng.processmaker.net

File: resources/js/processes-catalogue/components/ProcessTab.vue [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="mt-3">
-    <b-tabs content-class="text-style">
+    <b-tabs content-class="text-style" lazy>
```
Comment (fagubla) [inline resources/js/processes-catalogue/components/ProcessTab.vue:3]: Apply code style

**Review by fagubla — CHANGES_REQUESTED** [review]

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-966e8bebee.engk8s.processmaker.net

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-966e8bebee.engk8s.processmaker.net

**Review by fagubla — APPROVED** [review]

**Pull request merged**